File: DataAccess/Demand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Data
{
    public class Demand : BaseEntity
    {
        public string DescriptionEn { get; set; }
        public string DescriptionAr { get; set; }

        public int ComplaintID { get; set; }
        public virtual Complaint? Complaint { get; set; }
    }
}

File: UI/wwwroot/js/Demand.js
$("#AddDemand").on("click", function () {
    debugger;
    var DescriptionEnglish = $("#Description_en").val();
    var DescriptionArabic = $("#Description_ar").val();
    var RowNum = $("#DemandTabel tr").length;

    var Demand = {
        "DescriptionEn": DescriptionEnglish,
        "DescriptionAR": DescriptionArabic,
        "ID": RowNum,
        "ComplaintID": 0,
        "IsDeleted": false
    }

    $.ajax({
        url: '/Complaint/Demand',
        data: {
            demand: Demand
        },
        method: 'POST',
        type: 'JSON',
        success: function (result) {
            $("#DemandTabel").append(result);
        }

    });

});